docs/walkthroughs/running-with-secure-hdfs.md

```diff
@@ -0,0 +1,115 @@
+# Fully secure HDFS and Spark with Kerberos, SASL, and TLS. 
+This walkthrough will show how to run a version of the [TeraSort Benchmark](https://github.com/ehiggs/spark-terasort) with Spark in a fully secure deployment and Kerberos-secured HDFS as the storage backend. Specifically the following:
```

Comment:
"Specifically, we will use the following features:" (or something like that to avoid a fragment sentence)